src/app/core/config.py

```diff
@@ -81,5 +92,4 @@ class OtherSettings(BaseSettings):
 
 # 전역 설정 인스턴스
 settings = Settings()
-othersettings = OtherSettings()
-
+PascalCase = PascalCase()
```

Comment:
의미가 있는 이름이 되도록, 그리고 가능하면 클래스 이름에 대한 오버라이딩은 피해주시면 감사하겠습니다.
```suggestion
other_settings = OtherSettings()
```